app/views/payment_link_view.php

```diff
@@ -0,0 +1,54 @@
+<form method="post" action="payment_link_view">
+    <fieldset>
+        <?php
+        $paymentLink = $_SESSION['payment_link'];
+        ?>
+        <legend>Update Payment Link</legend>
+        <legend><b><?php echo $paymentLink->getId()?></b></legend>
+        <br/>
+        <pre><?php echo json_encode(json_decode($paymentLink->serialize()), JSON_PRETTY_PRINT);?></pre>
+
+        <div class="form-group row">
+            <label class="col-md-4 col-form-label">Expiration date</label>
+            <div class="col-md-3">
+                <select name="order[exp_year]"  class="form-control">
+                    <option value="">No End</option>
```

Comment:
This still shows `No end`. Design looks great on payment link creation view, same could be shown here